connexion/decorators/validation.py

```diff
@@ -73,6 +73,36 @@ def validate_parameter_list(request_params, spec_params):
     return request_params.difference(spec_params)
 
 
+def extend_with_nullable_support(validator_class):
+    """Add support for null values in body.
+
+    It adds property validator to given validator_class.
+
+    :param validator_class: validator to add nullable support
+    :type validator_class: jsonschema.IValidator
+    :return: new validator with added nullable support in properties
+    :rtype: jsonschema.IValidator
+    """
+    validate_properties = validator_class.VALIDATORS['properties']
+
+    def nullable_support(validator, properties, instance, schema):
+        null_properties = {}
+        for property_, subschema in six.iteritems(properties):
```

Comment:
It looks like this is breaking the build by not throwing the right exception when the function is given an integer as input.
See the travis build for the failure.